packages/migrations/package.json

```diff
@@ -10,13 +10,22 @@
     "scripts": {
         "build": "tsc -b",
         "build:ci": "yarn build",
-        "clean": "shx rm -rf lib",
+        "clean": "shx rm -rf lib ${npm_package_config_snapshot_name} *.zip",
```

Comment:
Can we make this `*.zip` a bit more specific? Worried a future dev might introduce a .zip and be very confused. Is there some suffix we could add to all the snapshots?